### PR TITLE
No bug - Fix the concordance search results

### PR DESCRIPTION
**Pull request by jotes**
TODO:
* [x] Add tests/double check tests
* [x] Update the commit message

### CODE REVIEW

File: pontoon/machinery/utils.py [hunk 1]
```diff
@@ -74,23 +75,27 @@ def get_concordance_search_data(text, locale):
...
     search_results = (
         base.models.TranslationMemoryEntry.objects.filter(search_query)
-        .annotate(
-            entity_pk=F("entity__pk"),
-            project_name=F("project__name"),
-            project_slug=F("project__slug"),
-        )
-        .values("source", "target", "entity_pk", "project_name", "project_slug")
+        .values("source", "target")
+        .annotate(project_names=ArrayAgg("project__name", distinct=True))
```
Comment (mathjazz) [inline pontoon/machinery/utils.py:79]: Nice thingy this `ArrayAgg`!

**Review by mathjazz — APPROVED** [review]
Nice job!

**Comment by jotes** [comment]
@mathjazz r?

**Review by mathjazz — APPROVED** [review]

**Pull request merged**